tools/istio-iptables/pkg/log/command.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+type Command struct {
+	Identifier string
+	Comment    string
+}
+
+var (
+	JumpInbound             = Command{"IPT01", "direct all traffic through ISTIO_INBOUND chain"}
+	JumpOutbound            = Command{"IPT02", "direct all traffic through ISTIO_OUTBOUND chain"}
+	ExcludeInboundPort      = Command{"IPT03", "exclude inbound port from capture"}
+	IncludeInboundPort      = Command{"IPT04", "include inbound port for capture"}
+	InboundCapture          = Command{"IPT05", "redirect inbound request to proxy"}
+	KubevirtCommand         = Command{"IPT06", "Kubevirt outbound redirect"}
+	ExcludeInterfaceCommand = Command{"IPT07", "Excluded interface"}
+	UndefinedCommand        = Command{"IPT00", ""}
+)
+
+var IDToCommand = map[string]Command{
```

Comment:
Why don't you just put the raw string or abbreviation in the NFLOG rule? Even abbr is better than number

Reply:
There is a very very small length allowed IIRC. I can look into it more, maybe its possible

Reply:
Feel free to ignore. No doubt this is far better than what we have nowadays.

Reply:
LOG has small limit, 64 for NFLOG though. lets do it